config.example.json

```diff
@@ -32,6 +32,9 @@
             }
         ]
     },
+    "constants": {
+        "defaultProfile": "https://yt3.ggpht.com/ytc/AMLnZu8gDqmPezdXMDI1k183oQeknA_V4ZDb6FQPo39PVg=s88-c-k-c0x00ffffff-no-rj"
```

Comment:
Thanks for adding it to the config, but since this will probably be used as the avatar url for production. Please make sure this is a permanent url (if this is a permanent url and I don't know it is, I'm sorry for the inconvenience I caused you)

Reply:
I'm unsure but if we just add the image to this repo we can be sure.